bundles/org.openhab.persistence.influxdb/README.md

```diff
@@ -93,6 +93,8 @@ temperature,item=tempBath,floor=firstfloor
 
 You can now easily select all temperatures of the firstfloor or the average temperature of the groundfloor.
 
+*Warning: Do **not** override the tag `item` within the metadata. This tag is used internally by OpenHAB and changing it will lead to problems querying the persisted datapoints.*
```

Comment:
I have a very small suggestion :slightly_smiling_face: :

```suggestion
*Warning: Do **not** override the tag `item` within the metadata. This tag is used internally by openHAB and changing it will lead to problems querying the persisted datapoints.*
```